src/network/netdev/l2tp-tunnel.h

```diff
@@ -0,0 +1,60 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include "in-addr-util.h"
+#include "netdev/netdev.h"
+
+typedef enum L2tpL2specType {
+        NETDEV_L2TP_L2SPECTYPE_NONE,
+        NETDEV_L2TP_L2SPECTYPE_DEFAULT,
+        _NETDEV_L2TP_L2SPECTYPE_MAX,
+        _NETDEV_L2TP_L2SPECTYPE_INVALID = -1,
+} L2tpL2specType;
```

Comment:
Why do not reuse kernel provided `l2tp_l2spec_type`?

Reply:
Need ```_NETDEV_L2TP_L2SPECTYPE_MAX``` and ```_NETDEV_L2TP_L2SPECTYPE_INVALID``` Now init to kernel defaults.